src/style.css

```diff
@@ -1,3 +1,68 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;700&family=Roboto:wght@400;500;700&display=swap");
+
 body {
   margin: 0;
+  --blue-color: #00acdc;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin-left: 50px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 60px;
+  justify-content: space-between;
+}
+
+.img {
+
+  height: 40px;
+}
+
+header a {
+  text-decoration: none;
+  color: #000;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  margin-right: 20px;
+  line-height: 60px;
+}
+
+a:hover {
+  color: var(--blue-color);
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  align-items: flex-end;
+  margin-right: 30px;
+
+}
+
+li:last-child {
+  margin-right: 0;
+}
+
+.is-active {
+  color: var(--blue-color);
+  position: relative;
```

Comment:
better to rewrite it using the class selectors instead of the tags
<img width="1146" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/c69183cc-fcda-4028-9262-ef9e4425080d">